R/reporter-progress.R

```diff
@@ -37,11 +39,13 @@ ProgressReporter <- R6::R6Class("ProgressReporter",
     initialize = function(show_praise = TRUE,
                           max_failures = getOption("testthat.progress.max_fails", 10L),
                           min_time = 0.1,
+                          update_interval = 0,
```

Comment:
Should the default be 0.1 here? Otherwise it is never throttled.

Reply:
Yeah, I changed the wrong value.